streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -152,6 +153,36 @@ public String toString() {
                 '}';
         }
     }
+    
+    public static class RankedClient<ID extends Comparable<? super ID>> implements Comparable<RankedClient<ID>> {
```

Comment:
@cadonna Does `RankedClient` sound good, or would you prefer `ClientAndRank` or...?

Reply:
I actually like `RankedClient` more. Thank you!